test/fixtures/rules/std_rule_cases/L025.yml

```diff
@@ -102,3 +102,19 @@ test_pass_subselect_uses_alias_3:
     SELECT col_1
     FROM table_a AS a
     WHERE NOT EXISTS (SELECT TRUE FROM table_b AS b WHERE a.col_4 = b.col_1)
+
+test_ansi_function_not_table_parameter:
+  fail_str: |
+    SELECT TO_JSON_STRING(t)
+    FROM my_table AS t
+  fix_str: |
+    SELECT TO_JSON_STRING(t)
+    FROM my_table
+
+test_bigquery_function_takes_table_parameter:
+  pass_str: |
+    SELECT TO_JSON_STRING(t)
+    FROM my_table AS t
+  configs:
+    core:
+      dialect: bigquery
```

Comment:
If you agree with above, should consider adding more test cases as described above.

Reply:
Will do.